packages/quantic/cypress/e2e/default-1/refine-toggle/refine-toggle.cypress.ts

```diff
@@ -145,6 +146,7 @@ describe('quantic-refine-toggle', () => {
             Expect.refineToggleTitleContains(defaultRefineToggleTitle);
             Expect.displayRefineToggleIcon(true);
             Expect.displayModal(false);
+            Expect.displayFacetManager(true);
```

Comment:
No need to check when Loading the page. checking this in the scope `when opening the refine modal` would make more sense and would be enough.

Reply:
yeah makes sense! thx 👍